### ISSUE TITLE
SDP ptime in microseconds instead of milliseconds. 

### ISSUE THREAD

**Issue by josafat-daysequerra** [martim01/pam#111]
The autogenerated SDP information reports ptime in microseconds. This seems unusual compared to other AES67 sources. 

Portion of the autogenerated SDP
```
a=ptime:1000
a=maxptime:1000
```

This makes the stream inaudible in my client software. 

Changing these lines to 1 allows the stream to playback normally. 
```
a=ptime:1
a=maxptime:1
```

The related file should be: `pambase/AES67ServerMediaSubsession.cpp` in lines:

```cpp
ss << "a=ptime:" << m_nPacketTime << "\r\n";
ss << "a=maxptime:" << m_nPacketTime << "\r\n";
```

**Comment by martim01** [comment]
Agreed - I've updated the code to populate the sdp with milliseconds rather than microseconds
